ONSdigital/ras-collection-instrument
# Removing database columns

# What and why?
The `items` and `status` columns in the `exercise` table needed to be removed, along with all references in the code.

# How to test?
Deploy, and make sure that the `items` and `status` columns are gone from the `exercise` table.

# Trello
[Card](https://trello.com/c/5wujnQMb)
